build_kernel.sh

```diff
@@ -10,62 +11,96 @@ OUTPUT_DIR=$DIR/output
 BOOT_IMG=./boot.img
 cd $DIR
 
+if [[ "$(uname)" == 'Darwin' ]]; then
```

Comment:
any way we can work around this? we're already building in a docker?

Reply:
the only way to go around the "Case-sensitive APFS volume" (just to build) is to clone the kernel inside the container build it there and just copy the output/boot.img and *.ko files. or make just "output" a volume.

the problem is that if you want to make changes, you don't have seamless visibility from macOS on the files inside the container. the `agnos-builder/agnos-kernel-sdm845` will still be empty (not cloned). and you need to work in a container or VM.

if you want to work on the kernel, like any other project on your computer, it should stay on a "Case-sensitive APFS volume". this is how I work on the kernel. the "Case-sensitive APFS volume" is very nicely implemented in macOS.